configs/xgboost/xgb_cpu_nvidia_config.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
You have 2 configs for CPU. Can we merge this into one?
Also, I prefer rename all things with "nvidia" in the name.